tests/pages/firefox/os/devices.py

```diff
@@ -0,0 +1,98 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait as Wait
+from selenium.webdriver.support.select import Select
+
+from ..base import FirefoxBasePage
+from ...page import PageRegion
+from ...regions.modal import Modal
+
+
+class DevicesPage(FirefoxBasePage):
+
+    _url = '{base_url}/{locale}/firefox/os/devices'
+
+    _location_select_locator = (By.ID, 'location')
+    _get_phone_button_locator = (By.CSS_SELECTOR, '.location-select-wrapper button')
+    _phone_thumbnail_locator = (By.CSS_SELECTOR, '#smartphones li:first-child > .device-thumbnail')
+    _phone_detail_locator = (By.CSS_SELECTOR, '.smartphone-detail-list .device-detail:first-child')
+    _tv_thumbnail_locator = (By.CSS_SELECTOR, '#tvs li:first-child > .device-thumbnail')
+    _tv_detail_locator = (By.CSS_SELECTOR, '.tvs-detail-list .device-detail:first-child')
+
+    def select_location(self, value):
+        assert self.is_button_disabled, 'Get phone button is already enabled'
+        el = self.selenium.find_element(*self._location_select_locator)
+        Select(el).select_by_visible_text(value)
+        Wait(self.selenium, self.timeout).until(lambda s: not self.is_button_disabled)
+
+    @property
+    def is_button_disabled(self):
+        el = self.selenium.find_element(*self._get_phone_button_locator)
+        return el.get_attribute('disabled') == 'true'
```

Comment:
I would suggest using the `is_enabled()` method provided by Selenium from the element, rather than checking the attribute ourselves. Also, the property name is not clear to be. Could we rename it to `is_get_a_phone_enabled`?